cli/src/commands/codegen/index.ts

```diff
@@ -0,0 +1,54 @@
+import { generateFromContext } from '@xata.io/codegen';
+import { mkdir, writeFile } from 'fs/promises';
+import fetch from 'node-fetch';
+import path, { dirname, extname, relative } from 'path';
+import { BaseCommand } from '../../base.js';
+
+const langauges: Record<string, 'javascript' | 'typescript'> = {
+  '.js': 'javascript',
+  '.mjs': 'javascript',
+  '.ts': 'typescript'
+};
+
+export default class Codegen extends BaseCommand {
+  static description = 'Generate code form the current schema';
+
+  static examples = [];
+
+  static flags = {};
+
+  static args = [{ name: 'output', description: 'The output file to generate', required: false }];
+
+  async run(): Promise<void> {
+    const { args } = await this.parse(Codegen);
+
+    const output = args.output || this.userConfig?.codegen?.output;
```

Comment:
[improvement for later]


If output is set by args, add an inquiry to set/update the config file.

If no config file yet, inquiry to initialize it with possible supported file names: .xatarc, xata.config.js....

Reply:
Yep. I've been thinking about this. I'd also like to add `declarations: true/false` and maybe in the near future plugins? One idea is to not allow arguments at all 🤔 with so many options and because if we always rely on the configuration file we have an ideal scenario to call the codegen when the schema changes, the user switches branches, etc.

I'm also thinking on adding `xata config get/set`. Similar to git config. Not sure if comsiconfig offers something like that though. I'll investigate.

But yeah, we need to provide a mechanism with prompts to configure those options if they are not set.